fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -280,7 +280,7 @@ protected Response exceptionResponse(Exception e, Status status) {
     }
 
     protected Response exceptionResponse(OperationOutcome oo, Status status) {
-        if (log.isLoggable(Level.FINE)) {
+        if (log.isLoggable(Level.FINE) && !Status.NOT_FOUND.equals(status)) {
```

Comment:
I don't understand how preventing the FINE-level log here prevents the stack trace described at https://github.com/IBM/FHIR/issues/1844#issuecomment-775277681

Reply:
That's why I assigned Robin - he didn't include any curl to replicate

I tried with the described resource that doesn't exist and it does not show up except in a trace.log (which is expectd\ed)